src/centralstore/incremental/NodeBlock.h

```diff
@@ -0,0 +1,73 @@
+/**
+Copyright 2020 JasminGraph Team
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+**/
+
+#include <cstring>
+#include <fstream>
+#include <iostream>  // for cout
+#include <list>
+#include <map>
+#include <string>
+
+#include "PropertyLink.h"
+
+class RelationBlock;  // Forward declaration
+
+#ifndef NODE_BLOCK
+#define NODE_BLOCK
+
+class NodeBlock {
+   private:
+    bool isDirected = false;
+
+   public:
+    unsigned int addr = 0;
+    std::string id = "";  // Node ID for this block ie: citation paper ID, Facebook accout ID, Twitter account ID etc
+    char usage = false;   // Whether this block is in use or not
+    char label[6] = {
+        0};  // Initialize with null chars label === ID if length(id) < 6 else ID will be store as a Node's property
```

Comment:
change `will be store` to `will be stored`

Reply:
Fixed in https://github.com/miyurud/jasminegraph/pull/115/commits/1d404d7d8fc79e4d908dcc39667a33b78beeaa26

Reply:
Done